NugetLightRequest.cs

```diff
@@ -1603,12 +1603,13 @@ into newGroup
                     else
                     {
                         SemanticVersion latest = new SemanticVersion("0.0.0");
-                        foreach (var p in pkgs) { 
-                            if (p.Version > latest) { 
+                        foreach (var p in pkgs) {
+                            // p.Version.SpecialVersion is the prerelease string after the patch number ex: for "1.0.0-alpha1" it would be "alpha1"
+                            if (p.Version > latest && String.IsNullOrWhiteSpace(p.Version.SpecialVersion) && !p.IsPrerelease) {
```

Comment:
This looks like if you specify -AllowPrerelease nothing will ever be returned, unless I am misreading? Don't we want

p.Version > latest && ( String.IsNullOrWhiteSpace(p.Version.SpecialVersion || p.IsPreRelease)

Reply:
the -AllowPrerelease case is actually handled right before this part on line 1594


                    }